src/Proj4.jl

```diff
@@ -101,11 +99,14 @@ end
 """
 Return true if the projection is a geocentric coordinate system
 """
-function is_gencent(proj::Projection)
+function is_geocent(proj::Projection)
     @assert proj.rep != C_NULL
     ccall((:pj_is_geocent, libproj), Cint, (Ptr{Void},), proj.rep) != 0
 end
 
+"Get projection definition string in the proj.4 plus format"
+proj4string(proj::Projection) = _get_def(proj)
```

Comment:
oh! that's cool :D I'll remove this line then (and the corresponding calls in the tests), and leave you to it when you merge?